src/TSTransformer/util/isMethod.ts

```diff
@@ -94,5 +95,6 @@ export function isMethod(
 	state: TransformState,
 	node: ts.PropertyAccessExpression | ts.ElementAccessExpression | ts.SignatureDeclarationBase,
 ): boolean {
+	validateNotAnyType(state, node);
```

Comment:
Why do we use `validateNotAnyType` here in particular?

Reply:
Because we need to know if we need to use . or : call syntax.